packages/logger/src/index.ts

```diff
@@ -1,5 +1,5 @@
-import debug from 'debug';
 import { format, Logger, createLogger, transports } from 'winston';
+import minimatch from 'minimatch';
```

Comment:
This is the package that allows glob matching. It is used for filterign `winston`'s namespaces.